code/game/machinery/syndicatebomb.dm

```diff
@@ -449,6 +455,33 @@
 	range_light = 4
 	range_flame = 2
 
+/obj/item/bombcore/emp
+	name = "EMP bomb core"
+	var/light_emp = 36
+	var/heavy_emp = 18
+	var/pulse_number = 1 //Since one EMP wont destroy anything other then consoles and IPCS, here is an option to have multiple pulses when dentonating. DO NOT USE THIS WITH REALLY LARGE AREAS
+	var/adminloged = FALSE //If it exploded once, don't do it again.
```

Comment:
```suggestion
	var/adminlogged = FALSE //If it exploded once, don't do it again.
```

Reply:
Need to change both variables, one moment